SpatialGDK/Extras/schema/migration_diagnostic.schema

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Please increment the RequireSetup file also

Reply:
Sure, done.